core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -846,6 +846,24 @@ secret_configs:
       - cluster: api.ci
         namespace: ci
         name: cluster-secrets-openstack-ppc64le
+  # cluster-secrets-openstack-OSUOSL
+  - from:
+      pull-secret:
+        bw_item: quay.io
+        attachment: pull-secret
+      clouds.yaml:
+        bw_item: ppc64le-osuosl-ci
+        attachment: cluster-secrets-osuosl-clouds
```

Comment:
Does this already exist?

Reply:
Yes.  This was merged https://github.com/openshift/release/pull/12442
and I did
oc create secret generic cluster-secrets-osuosl-clouds --from-file=/home/OpenShift/ppc64le-ci/root/director-ci-test/clouds-osuosl.yaml
Is this what you were looking for?